### PR TITLE
give cat rain more random spawn times

### PR DESCRIPTION
**Pull request by uku1928**
random.randint(1, 2) outputs either 1 or 2
random.uniform(1, 2) outputs any float between 1 and 2

### CODE REVIEW

**Comment by milena-kos** [comment]
eh, im honestly fine with just making it hard 1

File: main.py [hunk 1]
```diff
@@ -844,10 +844,13 @@ async def on_message(message):
             if cat_rains.get(str(message.channel.id), 0) != 0:
                 if cat_rains.get(str(message.channel.id), 0) > time.time():
                     times = [1, 2]
+                    decided_time = random.uniform(times[0], times[1])
                 else:
                     cat_rains[str(message.channel.id)] = 0
                     await message.channel.send("# :bangbang: this concludes the cat rain.")
-            decided_time = random.randint(times[0], times[1])
+                    decided_time = random.randint(times[0], times[1])
```
Comment (milena-kos) [inline main.py:851]: ?

Comment (uku1928) [reply main.py:851]: so that I'm covering the "cat rain ran out" and "no cat rain" options

**Comment by uku1928** [comment]
but yeah hard 1 is easier

**Pull request closed without merging**